### PR TITLE
Auto-merge minor/patch dependabot if tests pass

### PR DESCRIPTION
**Pull request by grepsedawk**
Example: https://github.com/grepsedawk/bonspiel.studio/pull/67

### CODE REVIEW

**Review by phallstrom — DISMISSED** [review]

**Comment by grepsedawk** [comment]
I don't know why circle isn't building for me.

**Comment by phallstrom** [comment]
Not sure either. This is strange though.. see the github pr and it says "Auto-merge enabled" and that it will squash and merge automatically.  Is that due to your new workflow? Or did some other settings get tweaked in this repo?

What's also strange is that I used to be able to merge things regardless of check status, but that's gone now.

**Comment by grepsedawk** [comment]
If you undo the automerge you can:
image

Automerge is a new feature, I just clicked automerge and walked away assuming the test suite would pass, or I'd come back to it later on
But I'll just merge this one, it might just be because no actual code changed besides the github action

File: .github/automerge.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Auto-merge
+
+on:
+  workflow_run:
+    types:
+      - completed
+    workflows:
+      - 'Lucky App CI'
```
Comment (brandondees) [inline .github/automerge.yml:8]: misnomer?

Comment (grepsedawk) [reply .github/automerge.yml:8]: Ooh, yea needs to be be the main ci workflow

**Pull request merged**